freemocap/core_processes/post_process_skeleton_data/process_single_camera_3d_data.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+import logging
+
+import numpy as np
+
+from freemocap.core_processes.capture_volume_calibration.triangulate_3d_data import save_mediapipe_skeleton_data_to_npy
+from freemocap.utilities.project_3d_data_to_z_plane import project_3d_data_to_z_plane
+
+logger = logging.getLogger(__name__)
+
+
+def process_single_camera_skeleton_data(input_image_data_frame_marker_xyz: np.ndarray, raw_data_folder_path: Path) \
+        -> [np.ndarray, np.ndarray]:
+
+    logger.info("Processing Single Camera Skeleton data...")
+
+    skeleton_reprojection_error_fr_mar = np.zeros(input_image_data_frame_marker_xyz.shape[0:2])
+
+    raw_skel3d_frame_marker_xyz = project_3d_data_to_z_plane(
```

Comment:
```suggestion
    if project_to_z_plane:
        input_image_data_frame_marker_xyz = project_3d_data_to_z_plane(
```

*(realizing this was too complex of a change to do via PR suggestions, but I'll finish it up and then try to avoid making the mistake later lol)